Ошибка: HTTPConnectionPool(host='kiev.ua', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001527026D570>: Failed to resolve 'kiev.ua' ([Errno 11001] getaddrinfo failed)"))